sortinghat/core/api.py

```diff
@@ -201,3 +202,47 @@ def delete_identity(uuid):
         uidentity.refresh_from_db()
 
     return uidentity
+
+
+@django.db.transaction.atomic
+def update_profile(uuid, **kwargs):
+    """Update unique identity profile.
+
+    This function allows to edit or update the profile information
+    of the unique identity identified by `uuid`.
+
+    The values to update are given as keyword arguments. The allowed
+    keys are listed below (other keywords will be ignored):
+
+       - 'name' : name of the unique identity
+       - 'email' : email address of the unique identity
+       - 'gender' : gender of the unique identity
+       - 'gender_acc' : gender accuracy (range of 0 to 100; by default,
```

Comment:
According to the tests, this should be:
`gender accuracy (range of 1 to 100; by default, set to 100)`

Reply:
Fixed!

Reply:
thanks!